frontend/src/components/PDAStackVisualiser/stackVisualiserStyle.ts

```diff
@@ -1,12 +1,12 @@
 import { styled } from 'goober'
 
-export const ContentContainer = styled('div')`
+export const ContentContainer = styled('div')<{$panelWidth?: number, $showLabWindow?: boolean}>`
   display: flex;         
   flex-direction: column; 
   align-items: center;   
   justify-content: flex-start; 
   bottom: 1rem;
-  left: 5rem;
+  left: ${p => p.$showLabWindow && p.$panelWidth ? `${p.$panelWidth + 110}px` : '5rem'}; 
```

Comment:
Is there a way to use more relative units? I remember there were a few UI bugs with components that had hard coded pixesl